sync/config/dashboard.yaml

```diff
@@ -26,8 +26,8 @@ archive: https://github.com/tektoncd/dashboard/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
-- name: release-v0.33.x
```

Comment:
I think we should have a new entry for `v034.x` instead of just an update to the existing one.

Reply:
I removed 0.33 as it's EOL and https://github.com/tektoncd/website/issues/481 suggests removing EOL releases from the website both to keep the list manageable and to steer users to supported versions.